compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/packaging/converters/URIConverter.java

```diff
@@ -127,35 +128,36 @@ public URI combine(URI s, String p) {
         String nightlyBuild = String.valueOf(RepoUtils.getBallerinaVersion().contains("SNAPSHOT"));
         EmbeddedExecutor executor = EmbeddedExecutorProvider.getInstance().getExecutor();
         for (String supportedPlatform : SUPPORTED_PLATFORMS) {
-            Optional<RuntimeException> runtimeException = executor.executeMainFunction("module_pull",
-                    remoteURI.toString(), modulePathInBaloCache.toString(), modulePath, proxy.getHost(),
-                    proxyPortAsString, proxy.getUserName(), proxy.getPassword(), RepoUtils.getTerminalWidth(),
-                    supportedVersionRange, String.valueOf(this.isBuild), nightlyBuild, IMPLEMENTATION_VERSION,
-                    supportedPlatform);
-            // Check if error has occurred or not.
-            if (runtimeException.isPresent()) {
-                String errorMessage = runtimeException.get().getMessage();
-                if (null != errorMessage && !"".equals(errorMessage.trim())) {
-                    // removing the error stack
-                    if (errorMessage.contains("\n\tat")) {
-                        errorMessage = errorMessage.substring(0, errorMessage.indexOf("\n\tat"));
-                    }
-    
-                    // if module already exists in home repository
-                    if (errorMessage.contains("module already exists in the home repository") && this.isBuild) {
-                        // Need to update the version of moduleID that was resolved by remote. But since the version
-                        // cannot be returned by the call done to module_pull.bal file we need to set the version from
-                        // the downloaded balo file.
-                        Patten patten = this.homeBaloRepo.calculate(moduleID);
-                        return patten.convertToSources(this.homeBaloRepo.getConverterInstance(), moduleID);
-                    }
-
-                    // check if the message is empty or not. Empty means module not found. Else some other error.
-                    // Log if it is some other error.
-                    if (!"".equals(errorMessage.replace("error: \t", "").trim())) {
-                        this.errStream.println(errorMessage.trim());
-                        return Stream.of();
-                    }
+            String errorMessage = "";
```

Comment:
Seems `proxyPortAsString` and `executor` is not needed anymore.

Reply:
removed `proxyPortAsString` and `executor`